### PR TITLE
Lenny/shooter

### PR DESCRIPTION
**Pull request by mihir-jain**
Added controllers for the shooter, so that we can change the power output and use the shooter. We merged it with the swerve code so that we can use the shooter while driving.

### CODE REVIEW

**Comment by noam987** [comment]
Wait until We merge The drive train fixes into master before we merge this.

**Review by noam987 — COMMENTED** [review]
Merge conflicts resolved, and Confirmed it does not interfere with the Swerve Code.

**Review by thetaspirit — APPROVED** [review]

**Comment by noam987** [comment]
DO NOT DELETE THE BRANCH UNTIL FRIDAY AND IT IS TESTED AND WORKS

**Pull request merged**
